plugins/grpc/stats/server_handler_test.go

```diff
@@ -357,3 +357,8 @@ func TestServerDefaultCollections(t *testing.T) {
 		}
 	}
 }
+
+func countAggregationValue(v int) *istats.CountAggregationValue {
```

Comment:
shouldn't this be "newCountAggregationValue" based on the code style?

Reply:
Done